pages/linux/dpkg-query.md

```diff
@@ -5,16 +5,20 @@
 
 - List all installed packages:
 
-`dpkg-query -l`
+`dpkg-query --list`
 
 - List installed packages matching a pattern:
 
-`dpkg-query -l '{{pattern}}'`
+`dpkg-query --list '{{pattern}}'`
```

Comment:
```suggestion
`dpkg-query --list '{{libc6*}}'`
```

I suggest use a concrete pattern example to show some of valid wildcards at once where pattern is expected. Example is taken from `man`.

Reply:
Wait, this is not part of the changes.

Reply:
Since I expanded the short options for prior examples to long form I'm happy to modify the examples to follow this convention.

It looks like the other translations of `dpkg-query` are also affected by these changes (using the long form of command names, is it worthwhile addressing these changes in the other translations? (In a separate PR if possible)

Reply:
> Wait, this is not part of the changes.

I am not sure whether I can close this conversation.